src/tool_operate.c

```diff
@@ -1057,6 +1057,58 @@ static CURLcode operate_do(struct GlobalConfig *global,
           my_setopt_str(curl, CURLOPT_PINNEDPUBLICKEY, config->pinnedpubkey);
 
         if(curlinfo->features & CURL_VERSION_SSL) {
+          /* Verify if config->cert is a PKCS#11 URI and set the engine
```

Comment:
nit: s/Verify/Check/ would seem like a better choice of words for what the code is doing

documenting in man page that files with names starting with `pkcs11:` need to be loaded using `./pkcs11:` syntax probably would be welcome

Reply:
Changed the comment following the suggestion.